Simulator/Vertices/Neuro/AllIZHNeurons.h

```diff
@@ -141,32 +141,39 @@ class AllIZHNeurons : public AllIFNeurons {
    ///  @param  allVerticesDevice   GPU address of the allNeurons struct on device memory.
    virtual void deleteNeuronDeviceStruct(void *allVerticesDevice) override;
 
-   ///  Copy all neurons' data from host to device.
-   ///
-   ///  @param  allVerticesDevice   GPU address of the allNeurons struct on device memory.
-   virtual void copyNeuronHostToDevice(void *allVerticesDevice) override;
+   // ///  Copy all neurons' data from host to device.
+   // ///
+   // ///  @param  allVerticesDevice   GPU address of the allNeurons struct on device memory.
+   // virtual void copyToDevice(void *allVerticesDevice) override;
```

Comment:
Similar questions/comment here. Cleanup comment formatting and why is method commented out?